lms/services/blackboard_api/_basic.py

```diff
@@ -75,7 +75,7 @@ def request(self, method, path):
 
         try:
             return self._send(method, url)
-        except (OAuth2TokenError, HTTPError):
+        except ExternalRequestError:
```

Comment:
`OAuth2TokenError` is an `ExternalRequestError` subclass so there's no need to catch both here. (It might be a little weird for `OAuth2TokenError` to subclass `ExternalRequestError` and maybe it shouldn't but that's out of scope for this PR.